transform-jsonata/README.md

```diff
@@ -20,6 +20,14 @@ pnpm dev
 # or pnpm dev-kubevirt to inject a different example transformation
 ```
 
+### Tracing
+
+```shell
+docker run --rm -d -p 9411:9411 openzipkin/zipkin
+
+pnpm dev-zipkin
```

Comment:
pnpm? What is that :sweat_smile:

Reply:
it's a more efficient alternative to npm CLI https://pnpm.io/